docs/testrunner-toolkit/configuration/espresso.md

```diff
@@ -291,3 +294,36 @@ __Example__:
 ```yaml
   annotation: com.android.buzz.MyAnnotation
 ```
+
+### `numShards`
+
+__Description__: `numShards` sets the number of separate shards to create. This option works for both emulators and RDC. For more shard tests, click [here](https://developer.android.com/training/testing/junit-runner#sharding-tests) to read more.
+
+__Type__: *int*
+
+__Example__:
+```yaml
+  numShards: 4
+```
+
+### `shardIndex`
+
+__Description__: `shardIndex` specifies which shard to run. This option works for both emulators and RDC. For more shard tests, click [here](https://developer.android.com/training/testing/junit-runner#sharding-tests) to read more.
+
+__Type__: *int*
+
+__Example__:
+```yaml
+  shardIndex: 1
+```
+
+### `clearPackageData`
+
+__Description__: Removes all shared state from your device's CPU and memory after each test.
```

Comment:
What does "shared state" mean in this context? I feel like this description is not a clear as it might be, but I need to understand better before I can help rewrite it.

Reply:
I think `shared state` is more like cache, background processes and maybe uncleaned data in CPU and memory. It's highly related to Operating System and the framework itself.

Reply:
Ok. That makes sense. Would it be "shared states" (plural), since you say "all", or is "shared state" a single status descriptor? So I could envision us saying either:
_Removes all shared states from the testing device's CPU and memory at the completion of each test._
or
_Wipes the shared state from the testing device's CPU and memory at the completion of each test._

Which one is more precise in this case?

Reply:
@tianfeng92 -- could you please specify which version in my comment is the correct statement? Right now, the sentence is not grammatically correct, but I am not sure which way is intended. Thanks!

Reply:
@sweeneyskirt-sl I'd vote 
```
 Removes all shared states from the testing device's CPU and memory at the completion of each test.
```
Thanks!